deps/v8/src/base/platform/time.cc

```diff
@@ -461,12 +461,6 @@ struct timeval Time::ToTimeval() const {
 TimeTicks TimeTicks::HighResolutionNow() {
   static bool verified_high_resolution_available = false;
```

Comment:
Could also just change this to true.

Reply:
Why is `verified_high_resolution_available` set to true in the current code? It doesn't seem to ever be referenced otherwise... is it just to be explicit?

Reply:
It is only used here, looks like a way to only print the warning once.